### ISSUE TITLE
[react/jsx-handler-names] False positive when prettier formatted multiline prop is used

### ISSUE THREAD

**Issue by yugandhar02** [yannickcr/eslint-plugin-react#2785]
I have a component with large nested tree.
When formatted with prettier it splits event handler name in multi line form due to limited print width something like below

```
class Foo extends React.PureComponent<Props> { 
  handleClick() { 
  }

  render() {
    return (
      <Bar 
        id={id},
        onClick={
          this
            .handleClick
         }
      />
    );
  }
}
```
It gives me error about invalid handler name even though the handler name is correct

`error      Handler function for onClick prop key must begin with 'handle'      react/jsx-handler-names`

**Comment by ljharb** [comment]
Although that prettier output is revolting, I agree it's a bug that the whitespace isn't taken into account.
